commands/ssh.go

```diff
@@ -24,7 +24,7 @@ func SSH() *cobra.Command {
 	addIntFlag(cmdSSH, doit.ArgDropletID, 0, "droplet id")
 	addStringFlag(cmdSSH, doit.ArgDropletName, "", "droplet name")
 	addStringFlag(cmdSSH, doit.ArgSSHUser, "root", "ssh user")
-	addStringSliceFlag(cmdSSH, doit.ArgSSHOption, nil, "ssh flag")
+	addStringFlag(cmdSSH, doit.ArgsSSHKeyPath, "", "path to private ssh key")
```

Comment:
Shouldn't this be `filepath.Join(usr.HomeDir, ".ssh", "id_rsa")` instead of a "".